src/ui/storyTellingServerUI.js

```diff
@@ -26,12 +30,16 @@ sjrk.storyTelling.getParameterByName = function (name, url) {
     return decodeURIComponent(results[2].replace(/\+/g, " "));
 };
 
-/* Loads a story View page and a particular story from a story ID from the query string
- * - "clientConfig": a collection of client config values consisting of
- *     - "theme": the current theme of the site
- *     - "baseTheme": the base theme of the site
- *     - "authoringEnabled": indicates whether story saving and editing are enabled
- * - "options": additional options to merge into the View page
+/**
+ * Loads a story View page and a particular story from a story ID from the query string
+ *
+ * @param {Object} clientConfig - a collection of client config values consisting of
```

Comment:
Another place that @typedef should be considered. An example is [here](https://github.com/GPII/universal/blob/master/gpii/node_modules/flowManager/src/PSPChannel.js#L177-L184).